wadas/domain/custom_classification_mode.py

```diff
@@ -60,33 +60,31 @@ def run(self):
             # Get image from motion detection notification
             # Timeout is set to 1 second to avoid blocking the thread
             try:
-                cur_img = img_queue.get(timeout=1)
+                cur_img = media_queue.get(timeout=1)
             except Empty:
                 cur_img = None
```

Comment:
This code is duplicated. You can put it in a function, on wadas.domain.camera.
It also helps encapsulating the code, without sharing further details on how it work (and it can be more easily be changed in future).